lib/grizzly.ex

```diff
@@ -107,6 +107,10 @@ defmodule Grizzly do
     send_command(Controller.conn(), command_module, command_opts)
   end
 
+  def send_command(%Node{conn: nil, id: 1}, command_module, command_opts) do
```

Comment:
Do you think that having the caller pass in `Grizzly.Controller` to `send_command` might be another solution? Maybe a helper like:

```elixir
defp send_command(%Node{id: 1}), do: Grizzly.send_command(Grizzly.Controller, command, args) 
```

Maybe this is fine, just trying to understand.

Reply:
I feel a bit uncomfortable with this as well but the call is made by Piston from interpreting device capabilities (e.g. the zwaveplus_info attribute applies to all devices that implement command class :zwaveplus_info). When the device is the controller, the zw_node.conn is nil and the send_command call would fail without this change. I can't at the moment think of a better solution.

Reply:
Correct me if I'm wrong but the only difference between what I did and what you propose is that I match on both the conn being nil and the id being 1, whereas you only match on id being 1. It appears that I'm wearing both a belt and suspenders :-)

Reply:
I guess my thoughts were is it possible for the caller, in this case, Pistion to just be able to figure out if it should pass the controller or a node? I guess I am trying to keep an implementation from another library from effecting what Grizzly supports in its API. That is Piston storing everything has a Node being the implementation detail.

I am okay with going ahead and adding this because I can see how it is useful, I am just trying to figure out where to draw the line between concerns here.

Reply:
Got you. Sorry for being a bit slow on the uptake. Looking to make the changes.

Reply:
I just tried putting the onus on Piston to detect conn == nil but it leads to a whole lot of breakage. Could we keep that change in Grizzly?